rest/model/build.go

```diff
@@ -15,26 +18,29 @@ var (
 
 // APIBuild is the model to be returned by the API whenever builds are fetched.
 type APIBuild struct {
-	Id                  APIString   `json:"_id"`
-	ProjectId           APIString   `json:"project_id"`
-	CreateTime          APITime     `json:"create_time"`
-	StartTime           APITime     `json:"start_time"`
-	FinishTime          APITime     `json:"finish_time"`
-	Version             APIString   `json:"version"`
-	Branch              APIString   `json:"branch"`
-	Revision            APIString   `json:"git_hash"`
-	BuildVariant        APIString   `json:"build_variant"`
-	Status              APIString   `json:"status"`
-	Activated           bool        `json:"activated"`
-	ActivatedBy         APIString   `json:"activated_by"`
-	ActivatedTime       APITime     `json:"activated_time"`
-	RevisionOrderNumber int         `json:"order"`
-	Tasks               []string    `json:"tasks"`
-	TimeTaken           APIDuration `json:"time_taken_ms"`
-	DisplayName         APIString   `json:"display_name"`
-	PredictedMakespan   APIDuration `json:"predicted_makespan_ms"`
-	ActualMakespan      APIDuration `json:"actual_makespan_ms"`
-	Origin              APIString   `json:"origin"`
+	Id                  APIString      `json:"_id"`
+	ProjectId           APIString      `json:"project_id"`
+	CreateTime          APITime        `json:"create_time"`
+	StartTime           APITime        `json:"start_time"`
+	FinishTime          APITime        `json:"finish_time"`
+	Version             APIString      `json:"version"`
+	Branch              APIString      `json:"branch"`
+	Revision            APIString      `json:"git_hash"`
+	BuildVariant        APIString      `json:"build_variant"`
+	Status              APIString      `json:"status"`
+	Activated           bool           `json:"activated"`
+	ActivatedBy         APIString      `json:"activated_by"`
+	ActivatedTime       APITime        `json:"activated_time"`
+	RevisionOrderNumber int            `json:"order"`
+	TaskCache           []APITaskCache `json:"task_cache"`
+	// Tasks is the build's task cache with just the names
+	Tasks             []string             `json:"tasks"`
```

Comment:
Then can we name the variable TaskCache?

Reply:
This one is not a task cache - it's just a list of task IDs

Reply:
oh look another misread.